net/FlatBuffers/Table.cs

```diff
@@ -65,6 +65,14 @@ protected int __vector(int offset)
             return offset + bb.GetInt(offset) + sizeof(int);  // data starts after the length
         }
 
+        // Get the data of a vector whoses offset is stored at "offset" in this object as an
+        // ArraySegment&lt;byte&gt;.
+        protected ArraySegment<byte> __vector_as_arraysegment(int offset) {
+          var pos = __vector(offset);
```

Comment:
Yes, it's correct. The code for __vector() is on line 65 in the code above. It returns the index of the first byte of the vector data, following the 4-byte length.